tests/dns/dns-rrtype/README.md

```diff
@@ -0,0 +1,5 @@
+Test the `dns.rrtype` value.
+
+The PCAP here was reused from ./tests/dns/dns-answer-name/dns-udp-request-with-answer.pcap
```

Comment:
I think this pcap is no longer used and needs to be updated here. Rest looks good! :)

Reply:
oops, thanks for pointing that out! I'll create a new PR.